network-api/networkapi/campaign/models.py

```diff
@@ -82,6 +82,26 @@ class Petition(models.Model):
         verbose_name="Mozilla Newsletter signup checkbox Google Form field",
     )
 
+    share_link = models.URLField(
+        max_length=1024,
+        help_text="Link that will be put in share button",
+        blank=True
+    )
+
+    share_link_text = models.CharField(
+        max_length=20,
+        help_text="Text content of the share button",
+        default="Share this",
+        blank=True
+    )
+
+    thank_you = models.CharField(
+        max_length=140,
+        help_text="Message to show after thanking people for signing",
+        default="Thank you for signing too!",
+        blank=False
```

Comment:
we can remove this, it's the default

Reply:
This prevents someone from deleting that and saving though, correct?